api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -1681,28 +1683,51 @@ private void enableNCDs() {
 
     private void enableMCHForms() {
 
-        // ToDo: Fix privileges for these 3 forms.  Not every user should have privileges.
+        if (config.getCountry().equals(ConfigDescriptor.Country.LIBERIA)) {
+
+            extensions.add(visitAction(Extensions.MCH_ANC_INTAKE_VISIT_ACTION,
+                    "ui.i18n.EncounterType.name." + EncounterTypes.ANC_INTAKE.uuid(),
+                    "fas fa-fw fa-gift",
+                    "link",
+                    enterStandardHtmlFormLink(PihCoreUtil.getFormResource("anc-initial.xml")),
+                    Privileges.TASK_EMR_ENTER_MCH.privilege(),
+                    and(sessionLocationHasTag(LocationTags.MCH_LOCATION),
+                            visitDoesNotHaveEncounterOfType(EncounterTypes.ANC_INTAKE),
+                            visitDoesNotHaveEncounterOfType(EncounterTypes.ANC_FOLLOWUP),
+                            and(patientIsFemale()))));
+
+            extensions.add(visitAction(Extensions.MCH_ANC_FOLLOWUP_VISIT_ACTION,
+                    "ui.i18n.EncounterType.name." + EncounterTypes.ANC_FOLLOWUP.uuid(),
+                    "fas fa-fw fa-gift",
+                    "link",
+                    enterStandardHtmlFormLink(PihCoreUtil.getFormResource("anc-followup.xml")),
+                    Privileges.TASK_EMR_ENTER_MCH.privilege(),
+                    and(sessionLocationHasTag(LocationTags.MCH_LOCATION),
+                            visitDoesNotHaveEncounterOfType(EncounterTypes.ANC_INTAKE),
+                            visitDoesNotHaveEncounterOfType(EncounterTypes.ANC_FOLLOWUP),
+                            and(patientIsFemale()))));
+
+
+            extensions.add(visitAction(Extensions.MCH_PEDS_ACTION,
+                    "ui.i18n.EncounterType.name." + LiberiaEncounterTypes.PEDS.uuid(),
+                    "fas fa-fw fa-gift",
+                    "link",
+                    enterStandardHtmlFormLink(PihCoreUtil.getFormResource("peds.xml")),
+                    Privileges.TASK_EMR_ENTER_MCH.privilege(),
+                    and(sessionLocationHasTag(LocationTags.MCH_LOCATION),
+                            and(patientIsFemale()))));
+        }
```

Comment:
You'll want an "else" here or otherwise I think you will get 2 sets of the forms in Liberia.

Reply:
You're right @mogoodrich. I don't know what's wrong with my head today🤕🤕🤕